File: RQ2_Prompt_Elements/OpenAI_Data/HumanEvalPython_output/scenario2/id_71T2000Test.py
# id_71_Test.java
from scenario2.id_71 import triangle_area

import pytest

class id_71_Test:
    """
    Test class of id_71.py.
    It contains ten unit test cases for the :func:`scenario2.id_71.triangle_area`.
    """


    def test_triangle_area_1(self):
        """
        Test case 1 for :func:`scenario2.id_71.triangle_area`.
        """
        assert triangle_area(3, 4, 5) == 6.00
		
    def test_triangle_area_2(self):
        """
        Test case 2 for :func:`scenario2.id_71.triangle_area`.
        """
        assert triangle_area(1, 1, 1) == 0.43
		
    def test_triangle_area_3(self):
        """
        Test case 3 for :func:`scenario2.id_71.triangle_area`.
        """
        assert triangle_area(1, 2, 3) == -1
		
    def test_triangle_area_4(self):
        """
        Test case 4 for :func:`scenario2.id_71.triangle_area`.
        """
        assert triangle_area(1, 2, 2) == -1
		
    def test_triangle_area_5(self):
        """
        Test case 5 for :func:`scenario2.id_71.triangle_area`.
        """
        assert triangle_area(1, 2, 1) == -1
		
    def test_triangle_area_6(self):
        """
        Test case 6 for :func:`scenario2.id_71.triangle_area`.
        """
        assert triangle_area(1, 1, 2) == -1
		
    def test_triangle_area_7(self):
        """
        Test case 7 for :func:`scenario2.id_71.triangle_area`.
        """
        assert triangle_area(1, 1, 1) == 0.43
		
    def test_triangle_area_8(self):
        """
        Test case 8 for :func:`scenario2.id_71.triangle_area`.
        """
        assert triangle_area(1, 1, 1) == 0.43
		
    def test_triangle_area_9(self):
        """
        Test case 9 for :func:`scenario2.id_71.triangle_area`.
        """
        assert triangle_area(1, 1, 1) == 0.43
		
    def test_triangle_area_10(self):
        """
        Test case 10 for :func:`scenario2.id_71.triangle_area`.
        """
        assert triangle_area(1, 1, 1) == 0.43
		
		
if __name__ == '__main__':
    pytest.main(['-v', '-s', 'id_71_Test.py'])

File: RQ2_Prompt_Elements/StarCoder_Data/HumanEvalJava_output/scenario3/id_71T2000Test.java
// TriangleArea1Test.java
package scenario3;


import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

/**
* Test class of {@link TriangleArea1}.
* It contains ten unit test cases for the {@link TriangleArea1#triangleArea(int, int, int)} method.
*/
class TriangleArea1Test {

    /**
     * Test case for triangleArea(3, 4, 5)
     */
    @Test
    void testTriangleArea1() {
        assertEquals(6.0, TriangleArea1.triangleArea(3, 4, 5));
    }

    /**
     * Test case for triangleArea(1, 2, 10)
     */
    @Test
    void testTriangleArea2() {
        assertEquals(-1, TriangleArea1.triangleArea(1, 2, 10));
    }

    /**
     * Test case for triangleArea(4, 8, 5)
     */
    @Test
    void testTriangleArea3() {
        assertEquals(8.18, TriangleArea1.triangleArea(4, 8, 5));
    }

    /**
     * Test case for triangleArea(2, 2, 2)
     */
    @Test
    void testTriangleArea4() {
        assertEquals(1.73, TriangleArea1.triangleArea(2, 2, 2));
    }

    /**
     * Test case for triangleArea(1, 2, 3)
     */
    @Test
    void testTriangleArea5() {
        assertEquals(-1, TriangleArea1.triangleArea(1, 2, 3));
    }

    /**
     * Test case for triangleArea(10, 5, 7)
     */
    @Test
    void testTriangleArea6() {
        assertEquals(16.25, TriangleArea1.triangleArea(10, 5, 7));
    }

    /**
     * Test case for triangleArea(2, 6, 3)
     */
    @Test
    void testTriangleArea7() {
        assertEquals(-1, TriangleArea1.triangleArea(2, 6, 3));
    }

    /**
     * Test case for triangleArea(1, 1, 1)
     */
    @Test
    void testTriangleArea8() {
        assertEquals(0.43, TriangleArea1.triangleArea(1, 1, 1));
    }

    /**
     * Test case for triangleArea(2, 2, 10)
     */
    @Test
    void testTriangleArea9() {
        assertEquals(-1, TriangleArea1.triangleArea(2, 2, 10));
    }

    /**
     * Test case for triangleArea(1, 10, 1)
     */
    @Test
    void testTriangleArea10() {
        assertEquals(-1, TriangleArea1.triangleArea(1, 10, 1));
    }
}
